app/graphql/mutations/media/update.rb

```diff
@@ -0,0 +1,10 @@
+module Mutations
+  module Media
+    class Update < UpdateMutation
+      graphql_name 'UpdateMediumMutation'
+      description 'Updates a new installation medium'
+
```

Comment:
I think we need `id` here, maybe `Mutations::UpdateMutation` is a good place to define it?
```suggestion
argument :id, ID, required: true
```

Reply:
:+1: